e2e-bitcoin-spv.yml

```diff
@@ -0,0 +1,25 @@
+relayer:
+  log-format: "auto" # (json|auto|console|logfmt)
+  log-level: "debug" # (debug|warn|error|panic|fatal)
+  retry-sleep-duration: 5s
+  max-retry-sleep-duration: 5m
+  netparams: simnet
+  cache-size: 2000
+  headers-chunk-size: 10
+  process-block-timeout: 20s
+btc:
+  no-client-tls: true
+  ca-file: $HOME/.btcd/rpc.cert
+  endpoint: localhost:18443
+  net-params: regtest
+  username: user
+  password: password
+  btc-backend: bitcoind # {btcd, bitcoind}
+  zmq-seq-endpoint: tcp://127.0.0.1:28331
+native:
+  rpc-endpoint: http://localhost:9797
+sui:
+  endpoint: http://127.0.0.1:9000
+  mnemonic: "bachelor domain magic seat income pledge now trip lumber rigid empower lesson"
+  lc_object_id: 0x747d8b46b49139d075132d412765f64480fcd1184e81a64610d180d8e7c48722
```

Comment:
lc_object_id not change for e2e test right?

Reply:
It does. Look at the `deploy_light_client.sh` we use `sed` to update it